third_party/blink/renderer/modules/mediasource/media_source.cc

```diff
@@ -57,6 +57,12 @@
 #include "third_party/blink/renderer/platform/wtf/functional.h"
 #include "third_party/blink/renderer/platform/wtf/text/wtf_string.h"
 
+// For BUILDFLAG(USE_STARBOARD_MEDIA)
+#include "build/buildflag.h"
+#if BUILDFLAG(IS_COBALT)
+#include "starboard/build/starboard_buildflags.h"
+#endif  // BUILDFLAG(IS_COBALT)
```

Comment:
I thought "build/buildflag.h" defined both IS_COBALT and USE_STARBOARD(_MEDIA) ?


(This is just a high level comment, not a review :)

Reply:
Right now, it defines just USE_STARBOARD and IS_COBALT—this will probably change as I said in my other comment